### ISSUE TITLE
requires auto-sklearn to plot kdes?

### ISSUE THREAD

**Issue by amueller** [janvanrijn/openml-pimp#4]
I need to install auto-sklearn to plot the KDEs. that seems weird?

**Comment by amueller** [comment]
And openmlstudy14... that requirements file would be really nice ;)

**Comment by amueller** [comment]
(and there's not even a pip package for openmlstudy14...)

**Comment by janvanrijn** [comment]
Just had a quick chat with Matthias, we both think that there should be a separate openml-python-utils package (or similar) to facilitate these kind of custom code. Will follow up on this. For the moment, does it work for you to just add this folder to the PYTHONPATH?

**Comment by amueller** [comment]
yes, that worked. I just wanted to point out that it's not great in terms of enabling others to run your code. Basically your requirements can't be complete given the way it's set up now.

Idea for a litmus-test: run the notebook on binder.
